src/Microsoft.Health.Dicom.Core.UnitTests/Serialization/ConfigurationJsonConverterTests.cs

```diff
@@ -58,7 +58,7 @@ public void GivenJson_WhenReading_ThenDeserialize()
         Assert.Equal("zero", actual["array:0"]);
         Assert.Equal("one", actual["array:1"]);
         Assert.Equal("two", actual["array:2"]);
-        Assert.Equal("true", actual["object:bool"]);
+        Assert.Equal("True", actual["object:bool"]);
```

Comment:
This serializes to a `string`, so it doesn't matter if it's upper-case or lower-case like it would for a json boolean